src/status_im/chat/screen.cljs

```diff
@@ -88,15 +88,13 @@
 (defn toolbar-action []
   (let [show-actions (subscribe [:chat-ui-props :show-actions?])]
     (fn []
-      (if @show-actions
-        [touchable-highlight
-         {:on-press #(dispatch [:set-chat-ui-props {:show-actions? false}])}
-         [view st/action
-          [vi/icon :icons/dropdown-up]]]
-        [touchable-highlight
-         {:on-press #(dispatch [:set-chat-ui-props {:show-actions? true}])}
-         [view st/action
-          [chat-icon]]]))))
+      [touchable-highlight
+       {:on-press #(dispatch [:set-chat-ui-props {:show-actions? (not @show-actions)}])
+        :accessibility-label :chat-menu}
+       [view st/action
+        (if @show-actions
+          [vi/icon :icons/dropdown-up]
+          [chat-icon])]])))
```

Comment:
I think we should leave logic alone unless it is necessary to change for the issue. Now there are two derefs which could be an unnecessary source of bugs, I believe this is why the conditional was at the top to begin with.